packages/malloy-interfaces/src/to_malloy.ts

```diff
@@ -379,6 +381,23 @@ function expressionToFragments(expression: Malloy.Expression): Fragment[] {
       ];
     case 'literal_value':
       return literalToFragments(expression.literal_value);
+    case 'moving_average':
+      return [
+        'avg_moving',
+        ...wrap(
```

Comment:
Pass `{ spaces: false }` here so it comes out as `avg_moving(x, 1)` rather than `avg_moving( x, 1 )`.